galleon-plugins/src/main/java/org/wildfly/galleon/plugin/config/WildFlyPackageTasksParser20.java

```diff
@@ -371,25 +378,23 @@ private void parseLineEndings(final XMLStreamReader reader, final WildFlyPackage
         throw ParsingUtils.endOfDocument(reader.getLocation());
     }
 
-    private void parseLineEnding(XMLStreamReader reader, final WildFlyPackageTasks.Builder builder, boolean windows) throws XMLStreamException {
+    private List<FileFilter> parseLineEnding(XMLStreamReader reader) throws XMLStreamException {
         if(reader.getAttributeCount() != 0) {
             throw ParsingUtils.unexpectedContent(reader);
         }
 
+        List<FileFilter> filters = Collections.emptyList();
         while (reader.hasNext()) {
             switch (reader.nextTag()) {
                 case XMLStreamConstants.END_ELEMENT: {
-                    return;
+                    return filters;
                 }
                 case XMLStreamConstants.START_ELEMENT: {
                     final Element element = Element.of(reader.getName());
+                    final FileFilter filter = parseFilter(reader);
                     switch (element) {
                         case FILTER:
-                            if(windows) {
-                                builder.addWindowsLineEndFilter(parseFilter(reader));
-                            } else {
-                                builder.addUnixLineEndFilter(parseFilter(reader));
-                            }
+                            CollectionUtils.add(filters, filter);
```

Comment:
Should be filters = CollectionUtils.add(filters, filter); otherwise the filters are lost.